R/pkg/R/functions.R

```diff
@@ -3652,3 +3652,43 @@ setMethod("posexplode",
             jc <- callJStatic("org.apache.spark.sql.functions", "posexplode", x@jc)
             column(jc)
           })
+
+#' collect_list
+#'
+#' Aggregate function: returns a list of objects with duplicates.
+#'
+#' @param x Column to compute on
+#'
+#' @rdname collect_list
+#' @name collect_list
+#' @family aggregate_functions
```

Comment:
re: my other comments, my preference would be full text without underscore like `@family aggregate functions`. This isn't a key or id and it shows up in generated doc text. Anyway, that's a bigger change all around.